drivers/sensor/bmi160/bmi160.c

```diff
@@ -927,6 +966,15 @@ int bmi160_init(const struct device *dev)
 		return -EIO;
 	}
 
+	pm_device_init_suspended(dev);
+
+	int ret = pm_device_runtime_enable(dev);
+
+	if (ret < 0 && ret != -ENOSYS) {
+		LOG_ERR("Failed to enabled runtime power management");
+		return -EIO;
+	}
+
```

Comment:
It is worth to mention that device is at this moment suspended. So, you have to call `pm_device_runtime_get` before effectively use it.

Reply:
It probably makes sense to call ```pm_device_runtime_get``` once in the device init so the behavior with or without PM is the same (the device is active) and the application can turn it off whenever it wants to. I'll do that.

Reply:
not sure ... if you do that here the device will never be suspended since the reference count will be unbalanced (unless that is exactly what you want).

The common operation is to call runtime_get / runtime_put when it is needed, if you can't afford the latency you have to keep it active.

Reply:
Yeah on second thought I think the init into a suspended state is the right behavior, it might need to be accounted for in some samples though if I start enabling PM by default in them (like the sensor shell)